pkg/agent/agent.go

```diff
@@ -216,14 +216,6 @@ func (k *Klocksmith) watchUpdateStatus(update func(s updateengine.Status)) {
 
 	go k.ue.ReceiveStatuses(ch, stop)
 
-	// synthesize first message
```

Comment:
`GetStatus()` doesn't actually trigger anything, it's a separate dbus call that gets the current status from updateengine. The information pulled from this call required to make sure that we don't get stuck if updateengine requested a restart when the update agent wasn't listening. It seems to me like there are only two ways we can get the desired behavior in this part; 1. The way it is in this pr or 2. unroll the loop once in `watchUpdateStatus`, but have the unrolled part pull the status information from `GetStatus` instead of from the channel passed to `ReceiveStatuses`.